docs/source/challenge-doc.rst

```diff
@@ -52,6 +52,14 @@ actuated by a control value between  :math:`[-1, 1]`, with -1 and 1 representing
 Observation Space
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
+**Observations Space**
+
+
+The obs_dict variable contains useful observations for completing the task. Please note that participants are encourage to 
+modify the obs_dict to customize their reward computations; yet values directly obtained outside the obs_dict, or directly from 
+the simulator might not be accessible in submissions.  
```

Comment:
This is confusing. Further clarify it. 
> The obs_dict variable contains useful features that are used to create observation vectors (configured via obs_keys) and for computing environment rewards (configures get_reward_dict(.) in via weighted_reward_keys). 

During training, participants are encouraged to add new keys to the obs_dict to further aid their reward computations. Note that the values obtained outside the provided obs_dict, or directly from the simulator might not be accessible during submission evaluations.

Reply:
Updated in [commit](https://github.com/MyoHub/myosuite/commit/9ff29e8cdacd0838ef4668f9ef6033618678e8d2), also include changes to available obs dict showed in today's meeting